test/integration/smoke/test_affinity_groups.py

```diff
@@ -240,6 +251,14 @@ def test_DeployVmAffinityGroup(self):
         self.assertEqual(host_of_vm1, host_of_vm2,
             msg="Both VMs of affinity group %s are on different hosts" % self.affinity.name)
 
+
+    def tearDown(self):
+        try:
+            # Clean up, terminate the created instance, volumes and snapshots
+            cleanup_resources(self.apiclient, self.cleanup)
+        except Exception as e:
+            raise Exception("Warning: Exception during cleanup : %s" % e)
```

Comment:
```suggestion
    def tearDown(self):
        super(<cls-name>, self).tearDown()
```
(replace \<cls-name\> !)